intrinsicsolver/elongateForms.go

```diff
@@ -0,0 +1,40 @@
+package intrinsicsolver
+
+import (
+	"strings"
+)
+
+/* Function elongateForms is investigating for short-form functions and changes them for their long equivalent. */
+func elongateForms(line *string, lines *[]string, idx int, name string) {
+	var currentFunctions int
+	pLines := *lines
+	totalFunctions := strings.Count(*line, "!")
+	for (currentFunctions != totalFunctions+1) && !strings.Contains(*line, "#!/bin/bash") && strings.Contains(*line, "!") {
```

Comment:
I don't like this one - let's look for _shebang_ only (`#!`) because path and everything else may look differently.

Reply:
Ok